xtext-website/documentation/310_eclipse_support.md

```diff
@@ -208,6 +208,21 @@ public void fixName(final Issue issue,
 }
 ```
 
+### Multi-Quickfixes
+Xtext also provides a Quickfix API for the definition of multi-quickfixes. It's implementation is quite similar to the quick fix above, but you need to call `acceptor.acceptMulti()` instead of `acceptor.accept()`.
+```java
+@Fix(IssueCodes.INVALID_FEATURE_NAME)
+public void fixFeatureName(Issue issue,
```

Comment:
would be nice to have an example for the second api too

Reply:
https://www.typefox.io/blog/xtext-2-13-0-released-semantic-editing-made-easy

Reply:
To keep the documentation as small as possible, I think it is enough to have only one example that makes the user aware of that feature. Once they start using that Multi-Quickfix API, they will search for further documentation/examples and will find the blog post above.